src/libs/ReportUtils.ts

```diff
@@ -3906,41 +3652,34 @@ function getTaskAssigneeChatOnyxData(accountID, assigneeEmail, assigneeAccountID
 
 /**
  * Returns an array of the participants Ids of a report
- *
- * @param {Object} report
- * @returns {Array}
  */
-function getParticipantsIDs(report) {
+function getParticipantsIDs(report: OnyxEntry<Report>): Array<number | undefined> {
     if (!report) {
         return [];
     }
 
-    const participants = report.participantAccountIDs || [];
+    const participants = report.participantAccountIDs ?? [];
 
     // Build participants list for IOU/expense reports
     if (isMoneyRequestReport(report)) {
-        return _.chain([report.managerID, report.ownerAccountID, ...participants])
-            .compact()
-            .uniq()
-            .value();
+        const onlyTruthyValues = [report.managerID, report.ownerAccountID, ...participants].filter(Boolean);
+        const onlyUnique = [...new Set([...onlyTruthyValues])];
+        return onlyUnique;
     }
     return participants;
 }
 
 /**
  * Return iou report action display message
- *
- * @param {Object} reportAction report action
- * @returns {String}
  */
-function getIOUReportActionDisplayMessage(reportAction) {
-    const originalMessage = _.get(reportAction, 'originalMessage', {});
+function getIOUReportActionDisplayMessage(reportAction: OnyxEntry<ReportAction>) {
+    const originalMessage = reportAction?.originalMessage as IOUMessage;
```

Comment:
Is this assertion necessary?

Reply:
yes, without that reportAction.originalMessage is type `unknown`